package.yaml

```diff
@@ -78,6 +78,7 @@ benchmarks:
 tests:
   graphql-parser-test:
     main:                Spec.hs
+    other-modules:       PrimitiveColumns
```

Comment:
In the hspec format, I don't think we have to include them like this. Do you think this is necessary?